flow360/component/flow360_params/solvers.py

```diff
@@ -229,13 +229,15 @@ class IncompressibleNavierStokesSolver(GenericFlowSolverSettings):
 class TurbulenceModelConstantsSA(Flow360BaseModel):
     """:class:`TurbulenceModelConstantsSA` class"""
 
+    model_type: Literal["SA"] = pd.Field("SA", alias="modelType", const=True)
```

Comment:
lets use full names, SpalartAllmarasConsts

Reply:
Fixed, renamed to SpalartAllmarasConsts